crates/kira/src/manager/backend/cpal/wasm.rs

```diff
@@ -23,6 +23,12 @@ pub struct CpalBackend {
 	state: State,
 }
 
+// WASM doesn't have threads yet so this is sound. The reason this can't be done properly is because
+// of cpal's use internally of the `wasm_bindgen::Closure` type which is only implemented for `dyn
+// FnMut` and not `dyn FnMut + Sync + Send`.
+unsafe impl Sync for CpalBackend {}
+unsafe impl Send for CpalBackend {}
+
```

Comment:
Got it working on WASM with this hack. Not sure if this is something you'd be comfortable with.

Reply:
I'm guessing once WASM does have threads, this won't be sound anymore? If so, I'm not comfortable with the hack.

Reply:
I'll look around and maybe ask on the Rust forum what kind of options we have here. There might be `#[cfg()]` flags that we can check to know for sure where it will be sound to implement `Sync` and `Send`.